package-lock.json

```diff
@@ -0,0 +1,3901 @@
+{
```

Comment:
I undesrstand this file was added unintentionally and should be removed.

Reply:
Deleted. Fixed in bf189e4